src/main/kotlin/br/com/WebApplication.kt

```diff
@@ -0,0 +1,11 @@
+package br.com
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+class WebApplication
+
+fun main(args: Array<String>) {
+	runApplication<WebApplication>(*args)
+}
```

Comment:
Create this project inside a project folder, like
https://github.com/vtripeno/integration_camel_apis